scripts/get-helm-3

```diff
@@ -57,13 +57,11 @@ initOS() {
 
 # runs the given command as root (detects if we are root already)
 runAsRoot() {
-  local CMD="$*"
-
-  if [ $EUID -ne 0 -a $USE_SUDO = "true" ]; then
-    CMD="sudo $CMD"
+  if [ $EUID -ne 0 -a "$USE_SUDO" = "true" ]; then
+    exec sudo "${@}"
```

Comment:
Could this be done without using `exec`? For example `"sudo \"$CMD\""`

Reply:
Sorry, yes of course we need to find another way, I was wrongly thinking, exec is not suitable here. I will recheck in the evening.

Reply:
I was changing it, and rebased to current master, but the changes do not show up, but are visible in my repo.. let's see. I ping you again, once I see the changes in the PR as well.

Reply:
@jdolitsky I had to change the destination branch forth and back, and now the latest changes show up. But now the label was not updated back and the DCO checks fails now for an old commit already in master; where the check was green after intially opening the PR.
Should I simply open a new PR to have a cleaner state?

Reply:
yes, its probably the easiest approach to close and re-open